src/Symfony/Component/Routing/Tests/Fixtures/controller_aware.yml

```diff
@@ -0,0 +1,9 @@
+app_homepage:
+    path: /
+    controller: AppBundle:Homepage:show
+
+app_blog:
+    path: /blog
+    controller: AppBundle:Homepage:show
```

Comment:
Any reason for this not to throw, but let the defaults override it?
IMHO, it'd be better to throw an exception when both are set, in order to avoid some potential WTF moments.